Makefile

```diff
@@ -89,8 +94,8 @@ OBJ += $(EXT_SRCS:.c=.o)
 ########################################
 CFLAGS = -std=gnu11 
 CFLAGS += -mcpu=cortex-m4 
-CFLAGS += -g3 
-CFLAGS += -O0 
+CFLAGS += -g3 #to remove
+CFLAGS += -O0 #to remove 
```

Comment:
This should not be removed... `-O0` forces no optimizations... we do not want to compiler doing funny things in this case.

Reply:
Please remove the `#to remove` comment.

Reply:
what about the -g3 ?
it adds symbol for debugging.

Reply:
Yes, this is for debugging. `-g3` provides extra marco expansions to the debugger. See the [GCC Docs](https://gcc.gnu.org/onlinedocs/gcc/Debugging-Options.html#Debugging-Options).

We can probably delete this eventually as well.

Reply:
done